core/src/test/java/greencity/service/impl/FactOfTheDayServiceImplTest.java

```diff
@@ -0,0 +1,174 @@
+package greencity.service.impl;
+
+import greencity.ModelUtils;
+import greencity.dto.PageableDto;
+import greencity.dto.factoftheday.FactOfTheDayDTO;
+import greencity.dto.factoftheday.FactOfTheDayPostDTO;
+import greencity.entity.FactOfTheDay;
+import greencity.exception.exceptions.NotFoundException;
+import greencity.exception.exceptions.NotUpdatedException;
+import greencity.repository.FactOfTheDayRepo;
+import greencity.service.LanguageService;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import static org.junit.Assert.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import static org.mockito.Mockito.*;
+import org.mockito.MockitoAnnotations;
+import org.modelmapper.ModelMapper;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
```

Comment:
You don't need SpringExtension as spring context is not used in the test, MockitoExtension would be enough.

Reply:
Changes done.